client.go

```diff
@@ -489,17 +489,32 @@ func (c *Client) websocket(operation string, secret string) (*websocket.Conn, er
 }
 
 func (c *Client) url(elem ...string) string {
+	// Normalize the URL
 	path := strings.Join(elem, "/")
+	entries := []string{}
```

Comment:
so your suggestion would fail with my current testcase which is:

c.url("containers//bin/bash")

So that's a single pre-joined element being passed to the function. Dropping all slashes makes it "/1.0/containersbinbash" which is very much invalid.